EPRN.UnitTests/API/PRNS/Services/PrnServiceTests.cs

```diff
@@ -124,16 +124,16 @@ public async Task SaveCheckYourAnswers_CallsService_WithComplete()
         {
             // arrange
             var id = 342;
-
+            
             // act
             await _prnService.SaveCheckYourAnswers(id);
 
             // assert
             _mockRepository.Verify(s =>
                 s.UpdatePrnStatus(
-                    It.Is<int>(p => p == id),
-                    It.Is<PrnStatus>(p => p == PrnStatus.CheckYourAnswersComplete),
-                    It.Is<string>(p => p == null)),
+                    It.IsAny<int>(),
```

Comment:
Since you're verifying a call has been made, you should know the parameters. I'm not sure why you removed all the It.Is for It.IsAny.

Reply:
Done.